utils/cert.js

```diff
@@ -1,11 +1,6 @@
-var IOTA = require('iota.lib.js')
 var crypto = require('crypto')
 
-var defaultNode = "http://node.deviceproof.org:14265";
-
-var iota = new IOTA({
-  provider: defaultNode
-});
+const iota = require('./iotaSetup.js');
```

Comment:
You shall mention how to generate `iotaSetup.js` along with top-level file `README.md`.